DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -78,35 +79,113 @@
           <ul class="nhsuk-header__navigation-list">
             @RenderSection("NavMenuItems", required: false)
             <li class="nhsuk-header__navigation-item">
-              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/Home?action=appselect">
-                Switch application
+              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/Home">
+                Welcome
                 <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
                   <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
                 </svg>
               </a>
             </li>
             <li class="nhsuk-header__navigation-item">
-              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/learningportal/help/LearningPortal.html" target="_blank">
-                Help
+              <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/Home#products">

```

Comment:
This is a last minute scope change (so sorry), but can we remove the products & learning content? 🙏